### PR TITLE
Add documentation for SmartTub integration

### PR DESCRIPTION
**Pull request by mdz**
## Proposed change

Add documentation for SmartTub integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/37775
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1744
- This PR fixes or closes issue: N/A

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by mdz** [comment]
The integration has been merged into dev

**Comment by frenck** [comment]
@mdz The PR was already marked as such, is there a problem with the marking?

**Comment by mdz** [comment]
I don't see any problem with the tags. Have a good day.

File: source/_integrations/smarttub.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- A SmartTub account (registration is not supported, you can use the SmartTub mobile app)
+
+## Configuration
+
+To add SmartTub to your installation:
+
+- Go to the integrations page in the Home Assistant frontend
+- Select `+` to add a new integration.
+- Add a new **SmartTub** integration.
+- Follow the steps shown to authenticate Home Assistant with your SmartTub account.
```
Comment (bdraco) [inline source/_integrations/smarttub.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/smarttub.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: SmartTub
+description: Instructions on how to integrate SmartTub into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 0.114
```
Comment (bdraco) [inline source/_integrations/smarttub.markdown]:
```suggestion
ha_release: 2021.3
```

**Review by bdraco — APPROVED** [review]
LGTM.  Thanks

**Pull request merged**